azure-pipelines.yml

```diff
@@ -12,4 +12,8 @@ steps:
     publishJUnitResults: true
     testResultsFiles: '**/TEST-*.xml'
     javaHomeOption: 'JDKVersion'
-    sonarQubeRunAnalysis: false
\ No newline at end of file
+    sonarQubeRunAnalysis: false
+
+- script: |
+    bash <(curl https://codecov.io/bash)
+  displayName: 'codecov'
```

Comment:
Can we give both more useful display names like "Run Build" and "Analyze Code" or something?

Reply:
Will do, I'm still testing though